driver.go

```diff
@@ -388,6 +388,11 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 		return nil, nil, err
 	}
 	createOpts.ContainerStorageConfig.Mounts = allMounts
+	allDevices := []spec.LinuxDevice{}
+	for _, device := range driverConfig.Devices {
+		allDevices = append(allDevices, spec.LinuxDevice{Path: device})
+	}
+	createOpts.ContainerStorageConfig.Devices = allDevices
```

Comment:
I wonder if we can pre-allocate the devices array based on the length of the config array and then insert using the loop index rather than appending?

Reply:
done